packages/lit-element/tsconfig.json

```diff
@@ -3,7 +3,11 @@
     "composite": true,
     "target": "es2020",
     "module": "es2015",
-    "lib": ["es2020", "DOM", "DOM.Iterable"],
+    "lib": [
```

Comment:
did you run `npm run format`?

Reply:
I did not, looks like I've got the wrong json formatter configured, fixed.